addons/im_livechat/controllers/main.py

```diff
@@ -146,9 +144,7 @@ def get_session(self, channel_id, previous_operator_id=None, chatbot_script_id=N
                 )
                 livechat_channel = livechat_channel.with_context(guest=guest)
                 request.update_context(guest=guest)
-            channel_vals = livechat_channel._get_livechat_discuss_channel_vals(
-                chatbot_script=chatbot_script, agent=agent
-            )
+            channel_vals = livechat_channel._get_livechat_discuss_channel_vals(**operator_info)
```

Comment:
same remark as the other methods: keep passing the known named parameters here and only spread the extra stuff